### DISCUSSION TITLE
Remove Device / Entity

### DISCUSSION THREAD

**Discussion by alutz33** [sbidy/wiz_light#159] [General]
I've installed all the WIZ lights via the Integration dialog. One of the lights was not setup correctly, so I tried to remove the device. But, there was no option to do so. So I deleted the entity to try and re-add the light from the integration dialog. However, I keep getting the error `Already configured. Only a single configuration possible.`  I also no longer see the old integration on the list. I tried renaming the new integration, but still receive the same error, so I assume it looks at the ip address? How can I delete this device and the associated IP to re-add?

**Comment by dfayne** [comment]
I'm having the same issue. Does anyone know a way to manually remove the entities? I have no idea where they live.

**Reply by tekdoodle** [reply]
This sounds similar to the issue #192 .
